### PR TITLE
Added pip3 updating and made installing composer optional

### PR DESCRIPTION
**Pull request by pahakalle**
Checks if you have pip3 as a command and updates it's packages.
Same command used for regular pip but with pip3

### CODE REVIEW

**Comment by pahakalle** [comment]
Method used is POSIX compatible: `command -v <the_command>`,
in this case `if [ -x "$(command -v pip3)" ];`

**Comment by pahakalle** [comment]
Also fixed #140 and now installing composer trough homebrew

File: mac-cli/tools/install [hunk 1]
```diff
@@ -136,12 +136,19 @@ main() {
...
-        curl -sS https://getcomposer.org/installer | php
-        echo "${GREEN}sudo mv composer.phar /usr/local/bin/\n${NC}"
-        sudo mv composer.phar /usr/local/bin/
-	fi
+        printf "${LIGHTGREEN}"
+        read -r -p "Would you like to install composer (https://getcomposer.org/)? (Yes / No)" response
+        case $response in
+            [yY][eE][sS]|[yY])
+                echo "Installing composer..."
+                echo "${GREEN}brew install composer\n${NC}"
```
Comment (pahakalle) [inline mac-cli/tools/install]: I don't see any down sides on using homebrew to install composer.

Comment (pahakalle) [reply mac-cli/tools/install]: so i changed it to use homebrew  instead of using `curl -sS https://getcomposer.org/installer | php`

**Comment by guarinogabriel** [comment]
Thanks so much for your contribution! 🎉

**Pull request merged**